editor.planx.uk/src/pages/FlowEditor/components/Team/queries/createAndAddUserToTeam.tsx

```diff
@@ -11,7 +15,7 @@ export const createAndAddUserToTeam = async (
   teamSlug: string,
 ) => {
   // NB: the user is hard-coded with the 'teamEditor' role for now
-  const response = (await client.mutate({
+  const response: CreateAndAddUserResponse = await client.mutate({
```

Comment:
FYI: `client.mutate` also accepts a generic which will automatically wrap the provided type with `FetchResult`, e.g. `client.mutate<MyType>`